app/services/onboarding.js

```diff
@@ -0,0 +1,56 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import { TOAST_OPTIONS } from '../constants/toast-options';
+
+export default class OnboardingService extends Service {
+  @service store;
+  @service login;
+  @service toast;
+
+  constructor() {
+    super(...arguments);
+    this.login.checkAuth();
```

Comment:
this `this.login.checkAuth();` might not be needed here

Reply:
removed